esterni/glossario/res/sections/dizionario.tex

```diff
@@ -108,6 +108,7 @@ \subsection{Proof of Concept} Prototipo software che presenta le caratteristiche
 \subsection{Proponente} Ente o azienda che propone il capitolato d’appalto per un progetto.
 \subsection{Protocollo} Insieme di regole e convenzioni necessarie a stabilire un dialogo fra due o più parti.
 \subsection{Publish/Subscribe} Protocollo di messaggistica in cui i mittenti di messaggi, chiamati publisher, non programmano di inviare il messaggio direttamente ai destinatari, chiamati subscriber, ma raccolgono i messaggi in classi senza sapere a chi siano destinati, eventualmente anche a nessuno. Similmente i subscriber possono esprimere interesse per una o più classi di messaggi senza sapere l'autore del messaggio.
+\subsection{Pull request} È una nostra richiesta, fatta all’autore originale di un sofware o di un documento, di includere le nostre personali modifiche al suo progetto.
```

Comment:
tradurre e sostituire
A pull request is a method of submitting contributions to an open development project. A pull request occurs when a developer asks for changes committed to an external repository to be considered for inclusion in a project’s main repository.